multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -14,6 +14,28 @@
 logger = logging.getLogger(AUTOMM)
 
 
+def from_dict(data):
+    """
+    Construct a dataframe (dummy) from a data dictionary, with the form {"image": ["img1.jpg", "img2.jpg", ...]}
+    Parameters
+    ----------
+    data
+        Dict containing the image paths
+    Returns
+    -------
+    a pandas DataFrame with columns "image", "rois", and "label".
+    """
+    d = {"image": [], "rois": [], "label": []}
+
+    for image in data["image"]:
+        d["image"].append(image)
+        # Dummy rois
+        d["rois"].append([[-1, -1, -1, -1, 0]])
```

Comment:
Add a TODO here, later we'll remove the dummy rois

Reply:
ok